### PR TITLE
Closes AgileVentures/MetPlus_tracker#264

### PR DESCRIPTION
**Pull request by patmbolger**

### CODE REVIEW

File: lib/task_manager/task_manager.rb [hunk 1]
```diff
@@ -48,10 +48,10 @@ def unschedule_event event_type, task, role_to_notify
 
     private
     ## Method that will create the task itself
-    def new_task task_type, owner, target_user = nil, target_job = nil, target_company = nil
+    def new_task task_type, owner, target_person = nil, target_job = nil, target_company = nil
       task = Task.new
       task.task_owner = owner
-      task.user = target_user
+      task.user = target_person.user if target_person
```
Comment (joaopapereira) [inline lib/task_manager/task_manager.rb]: use 
task.person = target_user
should be enough

File: spec/models/task_spec.rb [hunk 2]
```diff
@@ -166,30 +166,37 @@
     it 'job seeker user' do
       @task.target = @job_seeker
       expect(@task.target).to eq @job_seeker
+      expect(@task.user).to eq @job_seeker.user
```
Comment (joaopapereira) [inline spec/models/task_spec.rb]: we shouldnt use directly the user method but the person method, I will make user private for tasks

**Comment by patmbolger** [comment]
Updated per comments.

**Comment by joaopapereira** [comment]
Looks good going to merge

**Pull request merged**